opentracing_instrumentation/client_hooks/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015 Uber Technologies, Inc.
+# Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
I think you changed it in the wrong place, I meant the new file you're adding. This one can be changed to 2015-2017.

Reply:
:man_facepalming: fixed